go.mod

```diff
@@ -115,6 +115,8 @@ require (
 	modernc.org/sqlite v1.23.1
 )
 
+require github.com/bitnami/go-version v0.0.0-20231130082345-3f6796aa059b
```

Comment:
This is not the latest version: https://github.com/bitnami/go-version/commit/bb00604d650cf333cc3fcbc195983a1c3d73233e
This is fine?